CharacterCreation/02_Classes.txt

```diff
@@ -366,27 +367,83 @@ Skill Proficiencies:
+on your last turn. Furthermore you get a +12 to stealth(Dex) rolls made to lose 
+a tail or Piloting(Dex) rolls to make maneuvers in retreating.
+
+Wheelman:
+    
+    You have mastered the zen of the flow of vehicle combat. You don't just know
+when to break line of sight to avoid harm, but when to hold her steady so your
+teammates can light 'em up. You also have a concerning talent for ending enemy
+lives with the blunt force of your bumper.
+
+    Teammates firing from a vehicle that you are piloting get a +1 to accuracy
+rolls and negate any existing penalties from chaotic vehicle movement. If you
+attempt to make an attack by ramming, crashing, scraping or splattering, you 
+gain a +2 to hit with the attack roll and any damage to your vehicle from your
+attacks is reduced to three quarters. 
+
+    This Class Feat requires a Character to be level 10 and to have taken 
+Getaway Driver before they can take it.
+
+Boostin' Buggies:
+    
+    You have a deep familiarity with a variety of vehicle security systems and
+ignitions. 
+
+    You gain a +16 to skill checks with Slight of Hand(Dex), Hacking(Int), or
+Jerry Rig(Dex) to start and control a vehicle that didn't have the keys in it
+or you're otherwise not authorized to start. Vehicles that you steal won't 
+trigger car alarms or other security measures.
+
+Drive it Like You Stole it:
+
+    You have the habit of disabling the governer along with the car alarm every
+time you steal a vehicle. Pushing the vehicle to its limits is really hard on it,
+but you're sure insurance will cover it, whomever it belongs to. 
+
+    When piloting a vehicle that you've rolled a skill check to start, you have
+the option to "Push It". Every time you Push It, the vehicle takes a small 
+amount of damage at your DM's discretion. Pushing It can have the following 
+effects.
+
+* move 15%-20% faster than max speed (round to the most convinient whole number)
+* add +16 to a piloting check to maneuver
+* deal an extra 25% - 30% damage during a ramming, crashing, scraping or splattering attack (round to most convinient whole number)
+
+    To take this feat, a Character must be level 10 and have taken the "Boostin' 
+Buggies" Class Feat.
+
+Extensive training <Vehicle>:
+
 	Your complete knowledge of <Vehicle> makes your operation of it seem easy. 
-	Add	25 to any check concerning your operation, knowledge, or maintenance of the 
-	vehicle.
-	|
-	|
-	V
-	You are THE guru across the holonets for <Vehicle>. Add 50 onto a pertinent 
-	roll about the vehicle.
-======================================
+Add	25 to any check concerning your operation, knowledge, or maintenance of the 
+vehicle. A suggested list of vehicles to chose from is below, but it is not
+exhaustive.
+
+* car/van/truck
+* tank
+* small boat
+* large watercraft
+* helicopter
+* starfighter
+* warship
+
+<Vehicle> Mastery:
+
+	You are THE guru across the galaxy for <Vehicles>. Add 50 onto a pertinent 
+roll about the vehicle.
```

Comment:
This breaks my DC balance: 120 Extra-Crit + 60 Vehicle Skill + 50 Vehicle Mastery = 230 > 200 so pls nerf.